app/controllers/v0/preneeds/burial_forms_controller.rb

```diff
@@ -42,11 +40,8 @@ def burial_form_params
         )
       end
 
-      def validate!(form)
-        # Leave in for manual testing of new schemas before made available on Vets JSON Schema
-        # schema = JSON.parse(File.read(Settings.preneeds.burial_form_schema))
-        schema = VetsJsonSchema::SCHEMAS[FORM]
-        validation_errors = ::Preneeds::BurialForm.validate(schema, form)
```

Comment:
Refactor.  I can only guess this existed in this form to test schema changes during early development.  Can't think of another reason to have the `validate` method as a class method that takes a schema and form instead of just and instance method on the form object.

We have better ways to validate/test schemas against forms in our specs now, so I changed this validation task to an instance method